lib/Runtime/Base/ScriptContext.h

```diff
@@ -1424,7 +1424,20 @@ namespace Js
             return this->deferredBody;
         }
 
+    private:
+        Js::PropertyId emptyStringPropertyId;
     public:
+        Js::PropertyId GetEmptyStringPropertyId()
+        {
+            if (emptyStringPropertyId == Js::PropertyIds::_none)
+            {
+                Js::PropertyRecord const * propertyRecord;
```

Comment:
kind of good practice in general. what if for Example GetOrAddPropertyRecord not populate the out variable in all path.

---
In reply to: [131467416](https://github.com/Microsoft/ChakraCore/pull/3455#discussion_r131467416) [](ancestors = 131467416)